### PR TITLE
Bus mapping

### PR DESCRIPTION
**Pull request by telephon**
Continuous control for dirt synths and a handshake for tidal.

### CODE REVIEW

**Comment by yaxu** [comment]
It's possible that I'm doing something stupid, but this is what I get in the postwindow when I send a `/handshake` to port 57120 from tidal:
```
ERROR: Primitive '_NetAddr_SendMsg' failed.
Wrong type.
RECEIVER:
Instance of NetAddr {    (0x5648f8494958, gc=14, fmt=00, flg=00, set=02)
  instance variables [4]
    addr : Integer 2130706433
    port : nil
    hostname : "127.0.0.1"
    socket : nil
}
CALL STACK:
	MethodError:reportError
		arg this = <instance of PrimitiveFailedError>
	Nil:handleError
		arg this = nil
		arg error = <instance of PrimitiveFailedError>
	Thread:handleError
		arg this = <instance of Thread>
		arg error = <instance of PrimitiveFailedError>
	Object:throw
		arg this = <instance of PrimitiveFailedError>
	Object:primitiveFailed
		arg this = <instance of NetAddr>
	OSCFuncBothMessageMatcher:value
		arg this = <instance of OSCFuncBothMessageMatcher>
		arg msg = [*1]
		arg time = 493.813494396
		arg testAddr = <instance of NetAddr>
		arg testRecvPort = 57120
	OSCMessageDispatcher:value
		arg this = <instance of OSCMessageDispatcher>
		arg msg = [*1]
		arg time = 493.813494396
		arg addr = <instance of NetAddr>
		arg recvPort = 57120
	Main:recvOSCmessage
		arg this = <instance of Main>
		arg time = 493.813494396
		arg replyAddr = <instance of NetAddr>
		arg recvPort = 57120
		arg msg = [*1]
^^ The preceding error dump is for ERROR: Primitive '_NetAddr_SendMsg' failed.
Wrong type.
RECEIVER: a NetAddr(127.0.0.1, nil)
```

File: classes/SuperDirt.sc [hunk 1]
```diff
@@ -245,6 +257,12 @@ SuperDirt {
 
 		this.closeNetworkConnection;
 
+		netResponders.add(
+			OSCFunc({ |msg, time, tidalAddr|
+				senderAddr.sendMsg("/handshake_reply", *this.handshakeReplyData)
```
Comment (yaxu) [inline classes/SuperDirt.sc]: Maybe this needs to be `tidalAddr.sendMsg(..` ?

Comment (yaxu) [reply classes/SuperDirt.sc]: Yes I tried and it works with that change.

Comment (telephon) [reply classes/SuperDirt.sc]: Are you happy with the naming convention?

Comment (yaxu) [reply classes/SuperDirt.sc]: Yes the naming convention works for me.

**Pull request merged**